### DISCUSSION TITLE
How do I detect and record fast moving objects

### DISCUSSION THREAD

**Discussion by mannish-kumar** [blakeblackshear/frigate#3903] [Ask A Question]
My camera captures some portion of street and I want to capture fast moving objects like motorcycle moving at a speed of 20 to 30 KMPH.
I have turned on motorcycle tracking at camera level with min_score, threshold, min_area and max_area. Also defined the object at zone level.
Is there a way I can capture moving motorcycle?

**Comment by NickM-27** [comment]
It's not clear what the issue is, but you may want to try increasing `detect -> fps` as higher fps is better for detecting very fast moving objects. 

If you have the default 5 then maybe try 8

**Reply by mannish-kumar** [reply]
I want to record when a motorcycle passes in front of my home. I will increase fps and try it out.
